collector/lib/rust/test/src/lib.rs

```diff
@@ -0,0 +1,19 @@
+#[no_mangle]
+pub extern "C" fn external_add(left: u64, right: u64) -> u64 {
+    left + right
```

Comment:
```suggestion
    add(left, right)
```
This way would also test doing a call into a separate non-`extern "C"` method.

Reply:
Good idea - done!